ably/README.md

```diff
@@ -0,0 +1,65 @@
+## Overview
+The [Ably][1] platform is used to power realtime use cases such as multiplayer, chat, data synchronisation, data broadcast and notifications, for high-scale web and mobile applications around the world.
+
+Using our APIs, engineers are free to focus on building core functionality, rather than having to provision and maintain servers and cloud infrastructure.
+
+Using Ably’s Datadog Integration, you can:
+- Monitor realtime application metrics across your Ably applications
+- Track messages, channels and connection usage
+- Identify unexpected activity and troubleshoot potential issues
```

Comment:
Could you provide the value of monitoring this information? We like to see the following within the overview: 
- What does Ably do in 1-2 sentences
- What will customers get from this integration 
- What types of metrics?

Reply:
Updated in e20e3b965b44b64683573fa2cb3ca92045d2e600